<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DES [指导老师:覃中平, 编码:U200917937]</string>
  </property>
  <widget class="QLabel" name="mingwen_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>加密前明文</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="miwen_input_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="miwen_output_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="jiemi_output_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="set_key_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>加密密钥</string>
   </property>
  </widget>
  <widget class="QLabel" name="miwen_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>加密后密文</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;DES加解密展示程序&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="jiemi_key_input_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="jiemi_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>解密密钥</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mingwen_input_lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mingwen_output_show">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>解密后内容</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>225</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="encryption_btn">
    <property name="text">
     <string>加密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decryption_btn">
    <property name="text">
     <string>解密</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mingwen_input_lineEdit</tabstop>
  <tabstop>miwen_input_lineEdit</tabstop>
  <tabstop>encryption_btn</tabstop>
  <tabstop>miwen_output_lineEdit</tabstop>
  <tabstop>jiemi_key_input_lineEdit</tabstop>
  <tabstop>decryption_btn</tabstop>
  <tabstop>jiemi_output_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
